app/src/main/res/layout/activity_comments_layout.xml

```diff
@@ -72,13 +73,13 @@
                 android:background="@color/kds_white"
                 app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
                 tools:listitem="@layout/comment_card" />
-
         </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
         <ProgressBar
```

Comment:
@hadia we could use the same new cell I just created for the loading indicator, think about it!

Reply:
yes agree